src/recommendations/CardsList.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import { injectIntl } from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+
+import RecommendationCard from './RecommendationCard';
+
+const CardsList = (props) => {
+  const { recommendations } = props;
+
+  return (
+    <div className="d-flex card-list">
+
+      {
+      recommendations?.map((recommendation) => (
+        <RecommendationCard
+          key={recommendation.key}
+          recommendation={recommendation}
+        />
+      ))
+      }
+    </div>
```

Comment:
You should only show CardList if the recommendations exist, no need for optional chaining (? operator). 
```suggestion
    <div className="d-flex card-list">
      {recommendations.map((recommendation) => (
        <RecommendationCard
          key={recommendation.key}
          recommendation={recommendation}
        />
      ))}
    </div>
```